common/vpc_stack.py

```diff
@@ -0,0 +1,23 @@
+import config
+
+from aws_cdk import (Stack,
+    aws_ec2 as ec2,
+    Tags)
+
+from constructs import Construct
+
+STACK_ID = "dca-common"
```

Comment:
The generic template shouldn't reference the DCA project.

Reply:
fixed, thanks.